common/constants/api.js

```diff
@@ -5,21 +5,15 @@ import { formatUserData } from 'common/utils/formatters';
 export const getUserPromise = ({ token }) => get('auth/user/', { token });
 export const getCodeSchoolsPromise = () => get('api/v1/codeschools/');
 export const getTeamMembersPromise = () => get('api/v1/teamMembers/');
-
-/* Resources API */
-export const getResourcesPromise = parameters => {
-  const { page = 1 } = parameters;
-
-  return get(
-    'api/v1/resources/',
-    {
-      parameters: {
-        ...parameters,
-        page,
-      },
-    },
-    ResourcesAPI,
-  );
+export const getResourcesPromise = parameters =>
+  get('api/v1/resources/', { parameters }, ResourcesAPI);
+export const getResourcesBySearch = parameters =>
+  get('api/v1/search/', { parameters }, ResourcesAPI);
+export const getResourcesCategories = () => {
+  return get('api/v1/categories', {}, ResourcesAPI);
+};
+export const getResourcesLanguages = () => {
```

Comment:
Can we do `getResourcesBy`*

Reply:
So the reason I changed it was because we aren't really getting resources by language, rather that happens when we select a language and then get it with getResourcesPromise. When we use getResourcesLanguages we are populating the dropdowns with all the languages. i felt like it needed some sort of distinction. But of course, as you wish.

Reply:
The way I’m looking at it, I don’t actually even think multiple end points should have been made!

I would’ve presumed it would be /resources where query strings are the filter

Reply:
I see you’re point tho. Maybe getLanguageResources